### PR TITLE
Hardware wallet support baby step

### PR DESCRIPTION
**Pull request by szerintedmi**
Display list of accounts from Ledger and Trezor hardware wallets (on under-the-hood page)

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit ce40b0cb8ab1f8b3e7fe689b093643175707d904

https://app.netlify.com/sites/augmint/deploys/5b05be7f11b73b09267850db

File: src/containers/underthehood/components/HWWallets.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+import { ErrorPanel } from "components/MsgPanels";
+
+import HDKey from "hdkey";
+import ethUtil from "ethereumjs-util";
+
+import { TrezorConnect } from "utils/trezor-connectv4";
+import Transport from "@ledgerhq/hw-transport-u2f";
+import Eth from "@ledgerhq/hw-app-eth";
+
+// const handleGetAddress = async e => {
```
Comment (reyrodrigez) [inline src/containers/underthehood/components/HWWallets.js]: do you still need these comments?

Comment (szerintedmi) [reply src/containers/underthehood/components/HWWallets.js]: nope. going to remove

File: package.json [hunk 2]
```diff
@@ -35,6 +39,7 @@
     },
     "scripts": {
         "start": "react-scripts start",
+        "start:https": "HTTPS=true react-scripts start",
```
[the remark this answers is not in the document]
Comment (szerintedmi) [reply package.json]: would love to but browser complains with local metamask b/c that's on http ... Couldn't configure to https.

**Pull request merged**